verifiers/internal/gcb/verifier.go

```diff
@@ -40,9 +41,70 @@ func (v *GCBVerifier) VerifyArtifact(ctx context.Context,
 
 // VerifyImage verifies provenance for an OCI image.
 func (v *GCBVerifier) VerifyImage(ctx context.Context,
-	artifactImage string,
+	provenance []byte, artifactImage string,
 	provenanceOpts *options.ProvenanceOpts,
 	builderOpts *options.BuilderOpts,
 ) ([]byte, string, error) {
-	return nil, "todo", serrors.ErrorNotSupported
+	prov, err := ProvenanceFromBytes(provenance)
```

Comment:
will this fail if provenance is nil? would be good to test -- since generic containers don't need this

Reply:
generic containers cannot be verified via the GCB verifier. So I think this is fine until GCB supports provenance on the registry. Confirmed?